common/cah-model.h

```diff
@@ -0,0 +1,65 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <string>
+#include <chrono>
+#include <atomic>
+
+#include <librealsense2/rs.hpp>
+
+namespace rs2
+{
+    class ux_window;
+    class viewer_model;
+    class device_model;
+
+    // Helper class for setting , starting and checking a timeout.
+    // Inner units are [ms] , API units are seconds.
+    class timeout
```

Comment:
Is this class thread-safe?

Reply:
> Is this class thread-safe?

It was not intended to be passed between threads.
I can lock the 2 functions and the ctor and then it will be safe, we can later on move this to a LRS utility file..
or add a comment that it is not thread safe.